docs/serving/convert-deployment-to-knative-service.md

```diff
@@ -0,0 +1,71 @@
+# Converting a Kubernetes Deployment to a Knative Service
+
+This topic shows how to convert a Kubernetes Deployment to a Knative Service.
+
+## Determine if your workload is a good fit for Knative
+
+In general, if your Kubernetes workload is a good fit for Knative, you can remove a lot of your manifest to create a Knative Service.
+
+There are three aspects you need to consider:
+
+- All work done is triggered by HTTP.
+- The container is stateless. All state is stored elsewhere or can be re-created.
+- Your workload uses only Secret and ConfigMap volumes.
```

Comment:
I think this should also include autoscaling. The shown transformation would include scale the nginx pod for roughly every 100 concurrent HTTP requests, so you're saving even more YAML :)

Reply:
Good comment, thanks @markusthoemmes .

How about add a new section named as follow:

## Benefits

- Reduce footprint of the service instance as the instance will be scale to 0 when it became idle.
- Performance can be improved via build-in autoscaling for knative service.

Thoughts?